### DISCUSSION TITLE
MacOS ARM M1 & VS2022 error Assemblies

### DISCUSSION THREAD

**Discussion by chanox** [ikvmnet/ikvm#399] [Q&A]
i test and get error mesage IkvmGetReferenceAssemblies, IKVM not support osx arm?

**Comment by wasabii** [comment]
As of today the SDK tools have no OSX ARM64 support. IKVM runs fine, but the tools do not.

This is because the tools are currently locked at .NET Core 3.1, which has no OSX ARM64 support.

We are very close to fixing this, but not there yet.

**Reply by chanox** [reply]
thanks for information
